### DISCUSSION TITLE
Space lost 1 GB in a 7 GB USB stick on Ventoy exfat partition

### DISCUSSION THREAD

**Discussion by Menard01** [ventoy/Ventoy#2187] [Q&A]
and with only 6 files on it 
6 GB files on a 7 GB partition must let 1 GB space free, no ?

**Comment by uuaws2** [comment]
Well , the ventoy tool makes a partition for its system stuff so you dont need to leave a 1GB of free space , but if you want to you could let that space be for good measure .
